frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,84 +1,79 @@
+    'data-variants': product_variants_matrix,
+    'data-variant-change-trigger-identifier': variant_change_identifier
+  } do |f| %>
+  <div id="inside-product-cart-form" data-hook="inside_product_cart_form">
+    <% is_product_available_in_currency = @product.price_in(current_currency) && !@product.price.nil? %>
+    <% default_variant = default_variant(@variants) %>
+
+    <% if is_product_available_in_currency %>
+      <div id="product-price" class="mb-2 text-center text-md-left add-to-cart-form-price" data-hook="product_price">
+        <span class="price selling" content="<%= @product.price_in(current_currency).amount.to_d %>">
+          <%= display_price(default_variant) %>
+        </span>
+        <span content="<%= current_currency %>"></span>
       </div>
-    <% else %>
-      <%= hidden_field_tag "variant_id", @product.master.id %>
     <% end %>
 
-    <% if @product.price_in(current_currency) && !@product.price.nil? %>
-      <div data-hook="product_price" class="col-lg-5 mt-4">
-        <div id="product-price">
-          <h3 class="product-section-title"><%= Spree.t(:price) %></h3>
-          <div>
-            <span class="lead price selling" itemprop="price" content="<%= @product.price_in(current_currency).amount.to_d %>">
-              <%= display_price(@product) %>
-            </span>
-            <span itemprop="priceCurrency" content="<%= current_currency %>"></span>
-          </div>
+    <div class="text-center text-md-left add-to-cart-form-general-availability">AVAILABILITY:
+      <% if !is_product_available_in_currency %>
+        <%= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
+      <% elsif default_variant.in_stock? %>
+        <%= render 'cart_form_availability_templates', type: 'in-stock' %>
+      <% elsif default_variant.backorderable? %>
+        <%= render 'cart_form_availability_templates', type: 'backorderable' %>
+      <% else %>
+        <%= render 'cart_form_availability_templates', type: 'out-of-stock' %>
+      <% end %>
+    </div>
 
-          <% if @product.master.can_supply? %>
-            <link itemprop="availability" href="https://schema.org/InStock" />
-          <% elsif @product.variants.empty? %>
-            <span class="out-of-stock d-block mt-2">
-              <%= Spree.t(:out_of_stock) %>
-            </span>
-          <% end %>
-          <% if @product.backordered? %>
-            <div class="alert alert-warning" id="cart-backordered-info">
-              <%= Spree.t(:backordered_info) %>
-            </div>
-          <% end %>
+    <% if @product.variants_and_option_values(current_currency).any? %>
+      <hr>
+      <ul id="product-variants" class="product-variants">
+        <% used_variants_options(@variants).each_with_index do |option_type, index| %>
+          <li>
+            <% if option_type[:name] == "color" %>
+              <%= render "color_option_type", option_type: option_type, index: index %>
```

Comment:
same as cart_form_availability_templates, not sure we have this partial in a codebase

Reply:
https://github.com/spark-solutions/spree/blob/0d834689561bd19459c668c2d23fca61489bfff8/frontend/app/views/spree/products/_color_option_type.html.erb